src/pages/workspace/accounting/qbo/import/QuickbooksLocationsPage.tsx

```diff
@@ -20,8 +20,12 @@ function QuickbooksLocationsPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
-    const {syncLocations, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const {syncLocations, pendingFields, reimbursableExpensesExportDestination, nonReimbursableExpensesExportDestination} = policy?.connections?.quickbooksOnline?.config ?? {};
     const isSwitchOn = Boolean(syncLocations && syncLocations !== CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE);
+    const canImportLocation =
+        reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY &&
+        nonReimbursableExpensesExportDestination !== CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.VENDOR_BILL;
```

Comment:
NAB. The `locationsAdditionalDescription` copy does not tell you that you need to change the export company cards option

Reply:
Yeah, I agree that's NAB. We can improve it later to something like:

>QuickBooks Online does not support adding a location to vendor bills or checks. Update your export preferences to journal entry and credit/debit card if you'd like to import locations as tags.

Reply:
this is the current copy, right?
<img width="600" alt="Screenshot 2024-05-31 at 11 26 25 AM" src="https://github.com/Expensify/App/assets/98560306/73117a51-9f03-441c-9c69-22377950864b">

Reply:
^^ that's the old copy.